Sources/SwiftSyntax/Raw/RawSyntax.swift

```diff
@@ -219,7 +219,7 @@ extension RawSyntax {
   /// If the syntax tree did not contain a token and thus no trivia could be attached to it, `nil` is returned.
   /// - Parameters:
   ///   - leadingTrivia: The trivia to attach.
-  func withLeadingTrivia(_ leadingTrivia: Trivia) -> RawSyntax? {
+  func withLeadingTrivia(_ leadingTrivia: Trivia, arena: SyntaxArena) -> RawSyntax? {
```

Comment:
Is this actually needed? We're always using the one arena for `RawSyntax` anyway aren't we? You also removed these in your other PR, so there'll be a conflict (?)